docs/model-monitoring/monitoring-models-grafana.ipynb

```diff
@@ -62,68 +69,42 @@
+    "* **Average Latency** &mdash; the average latency over time in both 5-minute and 1-hour intervals (rolling)\n",
+    "* **Errors** &mdash; the number of errors over time\n",
+    "* **Predictions Count** &mdash; the number of predictions over time in both 5-minute and 1-hour intervals (rolling)\n",
+    "* **Custom Metrics** &mdash; custom metrics defined in the serving function\n",
     "\n",
     "## Model monitoring Applications dashboard\n",
-    "Model monitoring of deployed applications displays drift details in graphical format.\n",
-    "\n",
-    "* **Draft Status by Category** &mdash; the percentages of potential drift, no drift and detected drift\n",
-    "* **Average Drift Value Result** &mdash; the average of the potential and the detected drift values\n",
-    "* **Latest Result** &mdash; summary of the latest draft result \n",
-    "* **Application Summary** &mdash; table summary of the application results including the schedule time, the metric \n",
-    "name, metric kind, result status, and result numerical value. \n",
-    "* **Result Value by Time** &mdash; user-defined metric by time. This value is calculated based on user defined function \n",
-    "and represents a specific numerical result of the selected application by time.  \n",
-    "* **Drift Detection History** &mdash; drift status by time. Using this chart you can detect different types of drifts by time such as gradual drift or incremental drift. To learn more about different types of drift, see [Concept Drift](https://www.iguazio.com/glossary/concept-drift/). Results that\n",
-    "do not represent drift (\"irrelevant\") are filtered from this chart. \n",
+    "The Applications dashboard displays details of the selected metric.\n",
     "\n",
     "![applications](../_static/images/mm-applications.png)\n",
     "\n",
+    "* **Predictions** &mdash; the estimated number of predictions pushed to the selected model endpoint.\n",
+    "* **Metrics** &mdash; the number of times the selected metric was calculated for the selected model endpoint.\n",
+    "* **Average Value** &mdash; the average value of the selected metric for the selected model endpoint.\n",
+    "* **Latest Result** &mdash; the most recent metric calculation within the selected time range. If the selected metric is an application result, the chart also displays its status and type.\n",
+    "* **Metric Summary** &mdash; a table summary of the metric results including the start and end infer time, the metric kind (if relevant), and the numerical value, and status (if relevant). If the selected metric is an application result, the chart also displays its status, type, and extra data.\n",
+    "* **Metric Value by Time** &mdash; the metric value over time. The time value is based on the end of the infer time window.\n",
+    "\n",
+    "See more about  [Concept Drift](https://www.iguazio.com/glossary/concept-drift/). \n",
```

Comment:
I'm not sure why a reference to the concept drift page should be added here. While it might be relevant, we should find a more appropriate place for it.
```suggestion

```

Reply:
Deleted the reference altogether since later on there is a ref to the monitoring page, which references the concept drift page